src/matter/interaction/InteractionMessenger.ts

```diff
@@ -153,8 +153,13 @@ export class InteractionServerMessenger extends InteractionMessenger<MatterDevic
                 }
             }
         } catch (error: any) {
-            logger.error(error.stack ?? error);
-            await this.sendStatus(StatusCode.Failure);
+            if (error instanceof StatusResponseError) {
+                logger.info(`Sending status response ${error.code} for interaction error: ${error}`);
+                await this.sendStatus(error.code);
+            } else {
+                logger.error(error.stack ?? error);
```

Comment:
Let's avoid using error.stack :
1 _ Per this [doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/Stack), this is a non-standard property
2 _ There is no reason to use it only here versus every other logger.error(error) calls. So this behavior should be implemented inside the Logger to be consistent across the board
```suggestion
                logger.error(error);
```

Reply:
Sure (BTW: That was from you before, I did not change that at all. :-)) But yes in the meantime we also have handling in the logger for that